ChangeLog

```diff
@@ -1,5 +1,10 @@
 mbed TLS ChangeLog (Sorted per branch, date)
 
+= mbed TLS 2.x.x branch released xxxx-xx-xx
+
+Changes
+   * Support SSL testing in out-of-source builds using cmake. #1193
+
```

Comment:
Personally I'd call it 'SSL system and interoperability testing' rather than 'SSL testing', which doesn't quite describe it for me. Your choice.

Reply:
It's needed to do the full testing of the library, and specifically the part that needs it is about 99.9% of the testing of SSL. I think this is the right level of detail for the changelog. Before, you could test crypto and X509 functionality after a cmake out-of-tree build, but not the SSL functionality. Now you can test SSL as well.

Reply:
Ok, I can go with 'testing' but can it at least be 'TLS testing'?